### DISCUSSION TITLE
Mods get repositioned

### DISCUSSION THREAD

**Discussion by zzahkaboom24** [ppy/osu#15139] [Q&A] [answered]
Mods get repositioned when opening and closing the mod menu

Look at the Approach Different mod being at the top row when opening the mod menu, then going to the bottom row when closing it in the video below:

Video showing the issue: https://www.youtube.com/watch?v=CIUfC0yzJNs

**Comment by bdach** [comment, marked as answer]
the mod selection overlay is pending a redesign anyway, so might not be worth fixing.

**Reply by zzahkaboom24** [reply]
ok
